README.md

```diff
@@ -127,73 +105,16 @@ factory.address,gaslessAccount.address, recoverer, customLogic, walletIndex, byt
-3. In another terminal run `curl http://localhost:8090/getaddr` which will return a JSON with information of the running jsRelay Server, for example:
-```json
-{"relayWorkerAddress":"0xe722143177fe9c7c58057dc3d98d87f6c414dc95","relayManagerAddress":"0xe0820002dfaa69cbf8add6a738171e8eb0a5ee54",
-"relayHubAddress":"0x38bebd507aBC3D76B10d61f5C95668e1240D087F", "minGasPrice":"6000000000",
-"chainId":"31", "networkId":"31","ready":false,"version":"2.0.1"}
-```
-4. Send to relayManagerAddress at least 0.1 tRBTC to set it up
-5. Send to relayWorkerAddress at least 0.1 tRBTC to set it up
-6. Once both addresses have been funded, run `node dist/src/cli/commands/enveloping.js relayer-register --network <RSKJ_NODE_URL> --hub <RELAY_HUB_CONTRACT_ADDRESS> -m secret_mnemonic --from <ADDRESS>  --funds 1e17 --stake 3e17 --relayUrl <RELAY_URL>` where `secret_mnemonic` contains the path to a file with the mnemonic of the account to use during the relay server registration, `<ADDRESS>` is the account address associated to that mnemonic
-7.  Wait until the relay server prints a message saying `RELAY: READY`.
-
-## 5. Use MetaCoin <a id="c05"></a>
-
-Mint and send tokens without requiring RBTC for gas. Works on Regtest. 
-
-Try it: https://github.com/rsksmart/enveloping-metacoin
-
-## 6. Documentation <a id="c06"></a>
-
-For more detailed documentation, go [here](https://docs.google.com/document/d/1kan8xUFYgjWNozBfpkopn35P9E6IuRjC-PNhwcrQLN4/edit)
-
-## 7. Troubleshooting <a id="c07"></a>
-
-### 7.1. Running on macOS <a id="c07.1"></a>
-To run the project using Docker on a Mac, you must follow these steps or the scripts and web apps won't work.
-
-- Patch `readlink`
-The startup scripts assume that GNU's `readlink` command is available. But MacOS ships with BSD's `readlink`, which is incompatible with GNU's version. So you must patch `readlink`. This can be done as follows:
-
-```
-brew install coreutils
-ln -s /usr/local/bin/greadlink /usr/local/bin/readlink
-```
-
-After this step, you must make sure that your `PATH` variable gives priority to `/usr/local/bin` over `/usr/bin`. You can do it with `which readlink`, which should output `/usr/local/bin/readlink`. Alternatively try executing `readlink -f .`, if it works you're ok.
-
-### 7.2. Common errors when testing <a id="c07.2"></a>
-
-#### Running a test throws the Error: Cannot find module 'directory-to-the-project/enveloping/rsknode/test/Flows.test.ts'
+## Changelog
 
-Ensure that you are in the project's root directory and that the test's name has no typos
+### V2
 
-#### Running Flows.test.ts test throws the error: http://localhost:8090 => Error: local view call to 'relayCall()' reverted: view call to 'relayCall'..
+* In V2 the Relay Hub contract doesn't accept native-currency for payments.
```

Comment:
We can rephrase it like this: "In V2 the Relay Hub contract doesn't receive payments, the payment for the service (in tokens) is paid directly to the worker relaying the transaction on behalf of the user."